warp10/src/main/java/io/warp10/continuum/MacroFilterAuthenticationPlugin.java

```diff
@@ -103,7 +103,7 @@ public WriteToken extractWriteToken(String token) throws WarpScriptException {
         return null;
       }
 
```

Comment:
Apply same check as for read token at line 102
`if (null == MACRO || null == token || "".equals(token)) {`